com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.LookDev.cs

```diff
@@ -109,7 +109,7 @@ void IDataProvider.FirstInitScene(StageRuntimeInterface SRI)
             volumeGO.name = "StageVolume";
             Volume volume = volumeGO.AddComponent<Volume>();
             volume.isGlobal = true;
-            volume.priority = float.MaxValue;
```

Comment:
i am not sure about the intrication of this since float.maxValue != int.MaxValue.
since this is only for initialization I think it has low impact but could you confirm?

Reply:
We just want to have a top priority volume to override any kind of volume when using the LookDev. This one only apply to the "scene" of the LookDev. And the volume contains (normally) light and shadowing information.

Reply:
sounds good!